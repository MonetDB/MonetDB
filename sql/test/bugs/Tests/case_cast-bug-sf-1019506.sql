select 	cast(null as clob) as table_cat, 
	cast(s.name as clob) as table_schem, 
	cast(t.name as clob) as table_name, 
	case when k.name is null then cast(1 as smallint) else cast(0 as smallint) end as non_unique,
	cast(null as clob) as index_qualifier, 
	cast(i.name as clob) as index_name, 
	case i.type when 0 then cast(2 as smallint) else cast(3 as smallint) end as type, 
	cast(kc.nr as smallint) as ordinal_position,
	cast(c.name as clob) as column_name, 
	cast(null as char(1)) as asc_or_desc, 
	cast(null as integer) as cardinality, 
	cast(null as integer) as pages, 
	cast(null as clob) as filter_condition 

from 	sys.idxs i, 
	sys.schemas s, 
	sys.tables t, 
	sys.columns c, 
	sys.objects kc, 
	sys.keys k 

where 	i.table_id = t.id
and 	t.schema_id = s.id 
and 	i.id = kc.id 
and 	t.id = c.table_id 
and 	kc."name" = c.name 
and 	t.name = 'c1006309' 
and 	(k.type is null or k.type = 1) 
and 	s.name = 'sys'

order by non_unique, type, index_qualifier, index_name, ordinal_position;
