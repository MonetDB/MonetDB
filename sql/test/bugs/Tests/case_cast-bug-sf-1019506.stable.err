stderr of test 'case_cast-bug-sf-1019506` in directory 'sql/test/bugs` itself:


# 14:06:31 >  
# 14:06:31 >  Mtimeout -timeout 180 Mserver "--config=/ufs/niels/data/Linux-i686/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/niels/data/Linux-i686/lib/MonetDB" --set "gdk_dbfarm=/ufs/niels/data/Linux-i686/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/niels/data/Linux-i686/var/MonetDB/log" --set mapi_port=48987 --set sql_port=55051 --set monet_prompt= --trace "--dbname=mTests_src_test_bugs" "/net/stem.ins.cwi.nl/export/scratch1/niels/Linux-i686/build-sql/src/backends/monet4/sql_server.mil" ; echo ; echo Over..
# 14:06:31 >  


# 14:06:33 >  
# 14:06:33 >  Mtimeout -timeout 60 MapiClient --language=sql -u monetdb -P monetdb --host=stem --port=55051  < case_cast-bug-sf-1019506.sql
# 14:06:33 >  


# 14:06:33 >  
# 14:06:33 >  Done.
# 14:06:33 >  

