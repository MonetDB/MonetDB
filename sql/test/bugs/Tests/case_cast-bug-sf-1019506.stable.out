stdout of test 'case_cast-bug-sf-1019506` in directory 'sql/test/bugs` itself:


# 19:40:38 >  
# 19:40:38 >   mserver5 "--config=/ufs/niels/scratch/monetdb/Linux-x86_64/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=4 --set "monet_mod_path=/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/lib:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=31946 --set xrpc_port=44499 --set monet_prompt= --trace  "--dbname=mTests_src_test_bugs" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 19:40:38 >  

# MonetDB server v5.9.0, based on kernel v1.27.0
# Serving database 'mTests_src_test_bugs', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008- MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://alf.ins.cwi.nl:31946/
# MonetDB/SQL module v2.27.0 loaded

#function user.main():void;
#    clients.quit();
#end main;


# 08:38:12 >  
# 08:38:12 >  mclient -lsql -ftest -i -e --host=alf --port=34631 
# 08:38:12 >  

#select 	cast(null as clob) as table_cat, 
#	cast(s.name as clob) as table_schem, 
#	cast(t.name as clob) as table_name, 
#	case when k.name is null then cast(1 as smallint) else cast(0 as smallint) end as non_unique,
#	cast(null as clob) as index_qualifier, 
#	cast(i.name as clob) as index_name, 
#	case i.type when 0 then cast(2 as smallint) else cast(3 as smallint) end as type, 
#	cast(kc.nr as smallint) as ordinal_position,
#	cast(c.name as clob) as column_name, 
#	cast(null as char(1)) as asc_or_desc, 
#	cast(null as integer) as cardinality, 
% .,	.,	.,	sys.,	.,	.,	.,	.,	.,	.,	.,	.,	. # table_name
% table_cat,	table_schem,	table_name,	non_unique,	index_qualifier,	index_name,	type,	ordinal_position,	column_name,	asc_or_desc,	cardinality,	pages,	filter_condition # name
% clob,	varchar,	varchar,	smallint,	clob,	varchar,	smallint,	smallint,	varchar,	char,	int,	int,	clob # type
% 0,	0,	0,	1,	0,	0,	1,	1,	0,	1,	1,	1,	0 # length

# 11:55:41 >  
# 11:55:41 >  Done.
# 11:55:41 >  

